src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,28 @@
+.calendar {
+  padding: 10px;
+  width: #{$limit-width-of-row};
+  display: flex;
+  gap: $indent-of-days;
+  flex-wrap: wrap;
+
+  &__day {
+    box-sizing: border-box;
+    text-decoration: none;
+
+    width: $day-size;
+    height: $day-size;
+    background-color: #eee;
+    border: 1px solid black;
+
+    align-items: center;
```

Comment:
The 'align-items' and 'justify-content' properties are used with 'display: flex' or 'display: grid'. But here, you're using it with 'display: none'. It won't have any effect. It's better to move them to the selector where 'display: flex' or 'display: grid' is used.